_episodes/02-benchmarking.md

```diff
@@ -23,12 +23,15 @@ We will get into creating parallel programs in Python later. First let's see a s
 your system monitor, and run the following:
 
 ~~~python
+# A simple summation making use of numpy's vectorization:
```

Comment:
Try to avoid diminuative words like "simple"

Reply:
Thanks for the feedback, I'll try to remember it! (and change it here of course)